tools/static_check_containers

```diff
@@ -12,5 +12,5 @@ cre="${cre:-"podman"}"
 dockerfiles=(container/*/Dockerfile*)
 echo "# Static check of Dockerfiles"
 set -x
-$cre run -e 'HOME=/' --rm -v"$PWD":/repo -w /repo "$img" hadolint "${dockerfiles[@]}"
+$cre run -e 'HOME=/' --rm -v"$PWD":/repo:Z -w /repo "$img" hadolint "${dockerfiles[@]}"
```

Comment:
Maybe this should have a comment and a check to abort if $PWD is /home? or /usr? [The docs](https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label) say it renders the host unusable when one of those folders is used.

Reply:
Iirc podman refuses to relabel `/home`